governance/contributor-ladder.md

```diff
@@ -0,0 +1,111 @@
+As leadership roles naturally transition over time, it is worthwhile for you to consider taking up a leadership position within an established working group.
+Alternatively, you can contribute by proposing the formation of a new working group.
+To do so, create a GitHub issue where you outline the motivation, scope, goals, and deliverables of the proposed working group.
+Present this proposal during a TAG meeting to initiate a discussion, gather feedback, and garner support from fellow members.
+
+* Taking on more responsibility and ownership within TAG.
+* Leading a community group within TAG
+* Growing as a community leader
+* Conflict resolution
+
+## Nominated as Tech Lead
+
+To serve in the role as Tech Lead, you have to go through the [tech lead proposal process](https://github.com/cncf/tag-env-sustainability/blob/main/governance/tech-lead-proposal-process.md).
+Assuming the role of a technical lead within the Environmental Sustainability TAG places greater responsibilities on your shoulders as part of the leadership team.
+This roles provides you with the opportunity to actively support the leadership team by taking charge of a specific topic within the TAG.
+As an elected Tech Lead and a member of the leadership team, you will have the chance to delve into the broader aspects of leading the community group, gaining insights into the organizational structures within the CNCF. 
+Additionally, you will play a crucial role in facilitating discussions and resolving any issues that may arise between contributors, ensuring a harmonious and productive community environment.
+Your position as a technical lead allows you to contribute significantly to the overall success and cohesion of the Environmental Sustainability TAG.
+
+* Build partnerships between TAG and projects or other organizations.
+* Support and promote other community leaders
+
+## Nominated as Co-Chair
+
+To serve in the role of Co-Chair, you have to go through the [chair proposal process](https://github.com/cncf/tag-env-sustainability/blob/main/governance/chair-proposal-process.md).
+With your appointment as Co-Chair of TAG for Environmental Sustainability, you assume a key leadership position within the Group.
+Your will steer and evolve the community to achieve its goals and targets. 
+You coordinate between the TAG and the CNCF Technical Oversight Committee (TOC) and the overall CNCF ecosystem, ensuring effective communication and alignment of efforts.
+You will support and uplift other community leaders and provide guidance, mentorship, and opportunities for growth, you help cultivate a vibrant and empowered leadership team within the TAG.
+
+* Driving the continuous evolving of the TAG
+* Steering the TAG
+* Coordinating between the TAG and the TOC
+* Support and uplift other community leaders
+* Having joy seeing others progressing as leaders in the TAG
```

Comment:
I'd remove this last one.  It's not quantifiable.  

Maybe add, however, collaborating and steering new members into leading new positions on the tag, or something similar to that.